modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc

```diff
@@ -37,29 +28,30 @@ async function addPerson() {
   let client = await offlineClient.init();
 
   // Execute the mutation
-  client.query({
+  client.mutate({
       fetchPolicy: 'network-only',
```

Comment:
@finp I think this link might be your problem - probably left over from the query that was here. remove this and it should work.

Reply:
@finp - I tested this locally and it works, :+1: @StephenCoady 

I also modified it to write to the html page using:

 document.write(JSON.stringify(data))

![Screenshot from 2019-07-08 15-57-55](https://user-images.githubusercontent.com/19431386/60820561-926a0280-a199-11e9-9461-28623a2b2f51.png)

Reply:
@StephenCoady  thanks that was the trick. BTW we don't have a cli way to guide users thru this.
@jstaffor  I'm going with console cos a) we don't have a cli way of getting output b) we ask users later to use console https://docs.aerogear.org/aerogear/latest/ds-conflict.html#listening-to-conflicts

Reply:
:+1: